blog/2024-12-15-openhab-4-3-release.md

```diff
@@ -27,7 +27,7 @@ That being said, let's check out some statistics praising the activity of our co
 # Highlights
 
 As you might have already noticed in the activity statistics, there was much development on openHAB - too much to list everything.
-Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g. breaking changes).
+Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g., breaking changes).
```

Comment:
Do you really think the commata should be there?

Reply:
I tend to say yes, but I'm not a native speaker

Reply:
I have never seen that before, I would tend to say no.
Checking online it depends whether we use British or American English: https://grammarhow.com/eg-comma/
I am used to British English, that explains why I haven't used the comma. 

Can you please remove the commata after e.g.? IMO it looks kind of awkward to have .,

Reply:
```suggestion
Please refer to the [**release notes**](https://github.com/openhab/openhab-distro/releases/tag/4.3.0) to check out what's new and noteworthy (e.g. breaking changes).
```